src/crimson/osd/osd_operations/pg_advance_map.cc

```diff
@@ -133,4 +125,76 @@ seastar::future<> PGAdvanceMap::start()
   });
 }
 
+seastar::future<> PGAdvanceMap::split_pg(
+    std::set<std::pair<spg_t, epoch_t>> split_children_info,
+    cached_map_t next_map)
+{
+  using cached_map_t = OSDMapService::cached_map_t;
+  logger().debug("{}: start", *this);
```

Comment:
new debugging macro can be used in new code

Reply:
@Matan-B is it okay if I do this as a separate commit and use the debugging macro for the entire file (old code too)?

Reply:
Sure!